web/static/js/consent.js

```diff
@@ -3,8 +3,10 @@ var yesConsent = document.querySelectorAll(".yes-consent");
 
 [].forEach.call(noConsent, function(el){
   el.addEventListener("click", function (e) {
-    var index = e.target.id[e.target.id.length - 1];
-    document.getElementById("error-consent-" + index).classList.remove("dn");
+    if(e.target.name[e.target.name.length - 2] === "y") {
```

Comment:
The name is the redcap variable, which we won't necessarily control, with either '_y' or '_n' on the end, to denote whether or not it's required to be 'yes', so we can't check the whole name unfortunately